examples/helpers/cells.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright Schrodinger, LLC
+ */
+
+"use strict";
+
+const ExampleImage = require('./ExampleImage');
+const { Cell } = require('fixed-data-table-2');
+const React = require('react');
+const ReactTooltip = require('react-tooltip');
+
+class CollapseCell extends React.PureComponent {
+  render() {
+    const {data, rowIndex, columnKey, collapsedRows, callback, ...props} = this.props;
+    return (
+      <Cell {...props}>
+        <a onClick={() => callback(rowIndex)}>
+          {collapsedRows.has(rowIndex) ? '\u25BC' : '\u25BA'}
+        </a>
+      </Cell>
+    );
+  }
+};
+module.exports.CollapseCell = CollapseCell;
+
+class ColoredTextCell extends React.PureComponent {
+  render() {
+    const {data, rowIndex, columnKey, ...props} = this.props;
+    return (
+      <Cell {...props}>
+        {this.colorizeText(data.getObjectAt(rowIndex)[columnKey], rowIndex)}
+      </Cell>
+    );
+  }
+
+  colorizeText(str, index) {
+    let val, n = 0;
+    return str.split('').map((letter) => {
+      val = index * 70 + n++;
+      let color = 'hsl(' + val + ', 100%, 50%)';
+      return <span style={{color}} key={val}>{letter}</span>;
+    });
+  }
+};
+module.exports.ColoredTextCell = ColoredTextCell;
+
+class DateCell extends React.PureComponent {
+  render() {
+    const {data, rowIndex, columnKey, ...props} = this.props;
+    return (
+      <Cell {...props}>
+        {data.getObjectAt(rowIndex)[columnKey].toLocaleString()}
+      </Cell>
+    );
+  }
+};
+module.exports.DateCell = DateCell;
+
+class ImageCell extends React.PureComponent {
+  render() {
+    const {data, rowIndex, columnKey, ...props} = this.props;
+    return (
+      <ExampleImage
+        src={data.getObjectAt(rowIndex)[columnKey]}
+      />
+    );
+  }
+};
+module.exports.ImageCell = ImageCell;
+
+class LinkCell extends React.PureComponent {
+  render() {
+    const {data, rowIndex, columnKey, ...props} = this.props;
+    return (
+      <Cell {...props}>
+        <a href="#">{data.getObjectAt(rowIndex)[columnKey]}</a>
+      </Cell>
+    );
+  }
+};
+module.exports.LinkCell = LinkCell;
+
+module.exports.PagedCell = ({data, ...props}) => {
```

Comment:
Can we move this to be after PendingCell and move the exports to a new line, to follow the others